test/conformance/certificate/nonhttp01/run_test.go

```diff
@@ -19,11 +19,36 @@ limitations under the License.
 package nonhttp01
 
 import (
+	"context"
+	"flag"
+	"os"
 	"testing"
 
-	"knative.dev/networking/test/globals"
+	networkingtest "knative.dev/networking/test"
+	"knative.dev/networking/test/conformance"
+	"knative.dev/reconciler-test/pkg/test"
+	"knative.dev/reconciler-test/rigging/v2"
 )
 
-func TestCertificateConformance(t *testing.T) {
-	RunConformance(globals.NewT(t))
+var global rigging.GlobalEnvironment
+var nt = &networkingtest.T{}
+
+func TestMain(m *testing.M) {
+	global := test.NewGlobalEnvironment()
```

Comment:
Assuming this was intended

```suggestion
	global = test.NewGlobalEnvironment()
```